backup_and_restore/application_backup_and_restore/installing/installing-oadp-rosa.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="installing-oadp-rosa"]
+= Installing and configuring the OpenShift API for Data Protection (OADP) with Red Hat OADP Service for Amazon Web Services (ROSA)
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-rosa
+:installing-oadp-rosa:
+:credentials: cloud-credentials
+:provider: rosa
+
+toc::[]
+
+You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. Before installing OADP, you must set up role and policy credentials for OADP so that it can use the AWS API.
```

Comment:
```suggestion
You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. For more information about xref:../../../rosa/rosa_architecture/rosa-understanding.html[Understanding ROSA].

Before installing OADP, you must set up role and policy credentials for OADP so that it can use the AWS API.
```